src/store/index.js

```diff
@@ -0,0 +1,32 @@
+import { observable, action, makeObservable } from 'mobx';
+class UserStore {
+    jwt = undefined;
+    username = undefined;
+    isLogin = false;
+    constructor() {
+        this.jwt = localStorage.getItem('jwtToken')
+        this.username = localStorage.getItem('username')
+        this.isLogin = !!this.jwt;
+        makeObservable(this, {
+            jwt: observable,
+            username: observable,
+            isLogin: observable,
+            login: action,
+            logout: action,
+        });
+    }
+    login(jwt) {
+        this.jwt = jwt;
+        this.isLogin = !!jwt;
+    }
+
+
+    logout() {
+        this.jwt = undefined;
+        this.username = undefined;
```

Comment:
remove from localStorage as well
```suggestion
        this.username = undefined;
        localStorage.removeItem('jwtToken')
```